src/controllers/chatTribes.ts

```diff
@@ -366,10 +368,11 @@ export async function approveOrRejectMember(req, res) {
   if (
     !msgId ||
     !contactId ||
-    !(status === 'approved' || status === 'rejected')
+    !(req.params['status'] === 'approved' || req.params['status'] === 'rejected')
   ) {
     return failure(res, 'incorrect status')
   }
+  const status: ChatMemberStatus = req.params['status']
```

Comment:
was eslint complaining about having status defined higher up in the scope?

Reply:
no, status was defined earlier but i moved it down after the `if` because the type `ChatMemberStatus` cant be assigned before the check